test/specs/scale.logarithmic.tests.js

```diff
@@ -593,8 +593,8 @@ describe('Logarithmic Scale tests', function() {
 
 		var scale = chart.scales.y;
 		// Counts down because the lines are drawn top to bottom
-		expect(getLabels(scale)).toEqual([30, 20, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0]);
-		expect(scale.start).toEqual(0);
+		expect(getLabels(scale)).toEqual([30, 20, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0.9, 0.8, 0.7, 0.6, 0.5, 0.4, 0.3, 0.2, 0.1]);
```

Comment:
Should the `min` be `0.8` here?

Reply:
Its the `buildTicks` that alters it in this case:
`let tickVal = finiteOrDefault(generationOptions.min, Math.pow(10, Math.floor(log10(dataRange.min))));`

So if set in options to `0.8` that will be obeyed, but when coming from data, its going to start from the lover bound of that exp.